.github/workflows/js-benchmarks.yml

```diff
@@ -58,14 +58,28 @@ jobs:
           cd js-repl
           tar -xvzf ladybird-js-${{ matrix.os_name }}-${{ matrix.arch }}.tar.gz
 
-      - name: 'Run the JS benchmarks'
+      - name: 'Download Wasm repl artifact'
+        id: download-artifact
+        uses: dawidd6/action-download-artifact@v11
+        with:
+          run_id: ${{ github.event.workflow_run.id }}
+          name: ladybird-wasm-${{ matrix.package_type }}
+          path: wasm-repl
```

Comment:
You need to check `workflow_run` at the top of this file and see how the wasm build workflow ties into this; you'd need to start this workflow only when _both_ workflow have successfully completed.

If that's not possible, I wouldn't mind combining the two repl builds into a single workflow for this.